style.css

```diff
@@ -379,6 +456,499 @@ body {
 .butt:disabled {
   background: #fff;
   color: #5e6c84;
-  border: 1px solid #c1c7d0;
+  border: 0.0625rem solid #c1c7d0;
   cursor: not-allowed;
 }
+
+/* media queries
+for desktop */
+
+@media (min-width: 768px) and (max-width: 1029px) {
```

Comment:
## I'm sorry 🙏 
I made alll the adjustments  that you asked me. Unfortunately I'm not able to fix the display on a slightly bigger phone (like with a width of 540px ) because in the first week the requirements were to follow the Figma Template measures as closely as possible and the width specified was 375px, so I made my project with this size in mind. If I now try to use a responsive unit (as 100%) my webpage messes up completely and I would need to make changes on almost all the elements for the mobile version, so please just stick with the requirements for this project, it is a good review, thank you.